### PR TITLE
Custom Reports: fix table rendering

### PR DESCRIPTION
**Pull request by carkom**
In certain situations the table graph will not render properly and as a result columns will stop lining up properly. This PR should fix that issue.

### CODE REVIEW

**Comment by youngcw** [comment]
Nice.  I messed around and was not able to make a broken table.

File: packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTableList.tsx [hunk 1]
```diff
@@ -1,244 +1,95 @@
...
+    const rendered_row = renderItem({
+      item,
+      groupByItem,
+      mode,
+      style,
+      key,
+      monthsCount,
+    });
+
+    return rendered_row;
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTableList.tsx:49]: **🥜 nitpick:** ‏unnecessary variable assignment
```suggestion
    return renderItem({
      item,
      groupByItem,
      mode,
      style,
      key,
      monthsCount,
    });
```

**Review by MatissJanis — COMMENTED** [review]
A few small nitpicks (do whatever you prefer with them) and one more important question. Let me know what you think.

File: packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTable.tsx [hunk 2]
```diff
@@ -33,25 +46,57 @@ export function ReportTable({
...
-        scrollbarWidth: 'none',
-        '::-webkit-scrollbar': { display: 'none' },
         flex: 1,
+        flexDirection: 'row',
         outline: 'none',
         '& .animated .animated-row': { transition: '.25s transform' },
         ...style,
       }}
       tabIndex={1}
+      data-testid="table"
```
[the remark this answers is not in the document]
Comment (carkom) [reply packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTable.tsx:76]: we dont.

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**